src/accessor/mod.rs

```diff
@@ -119,8 +119,10 @@ impl<'a> Accessor<'a> {
     }
 
     /// Returns the offset relative to the start of the parent buffer view in bytes.
-    pub fn offset(&self) -> usize {
-        self.json.byte_offset as usize
+    ///
+    /// This may be `None` if the corresponding accessor is sparse.
+    pub fn offset(&self) -> Option<usize> {
```

Comment:
This will be a semver breaking change. I'm wondering whether we can return zero instead if it's `None` or whether we should bump the major version. Since https://github.com/gltf-rs/gltf/issues/380 I'm tempted to move to version 2 and start taking semver more seriously as discussed in https://github.com/gltf-rs/gltf/issues/241.

Reply:
I think that's fine by me. But this PR already breaks semver for gltf-json. Let me know how you would like to proceed and I'll make the adjustment.

Reply:
@alteous any decision on this? I'm happy to make the change, otherwise feel free to make changes to this PR directly.

Reply:
I'm not so concerned about breaking semver for gltf-json. It's not the first time this has happened. As long as the top-level gltf crate is unaffected. I suggest we go with the 'return zero' approach for now.

> Edit: I should add, that if we do return zero, we should probably still change it to return Option<usize> for the next version.

I agree with you. This is something that could be missed easily. I'm not how that could be prevented easily though.